src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin square ($size) {
+  width: $size;
+  height: $size;
+  position: absolute;
+  border-radius: 50%;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@mixin line ($width, $height) {
```

Comment:
Add a comment explaining the purpose of the `line` mixin. For example: `// Creates a line shape with given width and height, and sets the transform origin to the bottom ⬛`.